src/Libraries/CoreNodeModels/DropDown.cs

```diff
@@ -97,7 +97,7 @@ protected DSDropDownBase(string outputName)
         }
 
         [JsonConstructor]
-        protected DSDropDownBase(IEnumerable<PortModel> inPorts, IEnumerable<PortModel> outPorts) : base(inPorts, outPorts)
+        protected DSDropDownBase(string outputName, IEnumerable<PortModel> inPorts, IEnumerable<PortModel> outPorts) : base(inPorts, outPorts)
```

Comment:
whats this change about?

Reply:
if you look at the non-json constructor above it takes a single string parameter for the `outputName`.  The json constructor was previously created in another PR but this parameter was missing and is required to correctly pass the `outputName` from the child to the base constructor